src/Facets/DeBridgeDlnFacet.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { ILiFi } from "../Interfaces/ILiFi.sol";
+import { LibDiamond } from "../Libraries/LibDiamond.sol";
+import { LibAsset, IERC20 } from "../Libraries/LibAsset.sol";
+import { LibSwap } from "../Libraries/LibSwap.sol";
+import { ReentrancyGuard } from "../Helpers/ReentrancyGuard.sol";
+import { SwapperV2 } from "../Helpers/SwapperV2.sol";
+import { Validatable } from "../Helpers/Validatable.sol";
+import { IDlnSource } from "../Interfaces/IDlnSource.sol";
+
+/// @title DeBridgeDLN Facet
+/// @author LI.FI (https://li.fi)
+/// @notice Provides functionality for bridging through DeBridge DLN
+/// @custom:version 1.0.0
+contract DeBridgeDlnFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+
+    address internal constant NON_EVM_ADDRESS =
+        0x11f111f111f111F111f111f111F111f111f111F1;
+    IDlnSource public immutable dlnSource;
+
+    /// Types ///
+
+    // struct BridgeData {
+    //     bytes32 transactionId;
+    //     string bridge;
+    //     string integrator;
+    //     address referrer;
+    //     address sendingAssetId;
+    //     address receiver;
+    //     uint256 minAmount;
+    //     uint256 destinationChainId;
+    //     bool hasSourceSwaps;
+    //     bool hasDestinationCall;
+    // }
+
+    /// @dev Optional bridge specific struct
```

Comment:
are they all optional?? If yes, a description in which case they are needed would be good

and param descriptions are missing